CONTRIBUTING.md

```diff
@@ -18,19 +20,36 @@ Thank you for your contributing to SmartBear MCP!
 ### Repository Structure
 
 ```
-├── src/api-hub/      # API Hub integration
-├── src/bugsnag/      # BugSnag integration
-├── src/pactflow/     # PactFlow integration
-├── src/reflect/      # Reflect integration
-├── src/common/       # Shared utilities and types
-├── src/tests/        # Test files
-├── docs/             # Documentation
-└── scripts/          # Build and utility scripts
+├── src/<product-name>/      # Specific SmartBear product integration
+├── src/common/              # Shared utilities and types
+├── src/tests/               # Test files
+├── docs/                    # Documentation for developer.smartbear.com site
+└── scripts/                 # Build and utility scripts
 ```
 
+### Branching Strategy
+
+Releases are automatically deployed on merge to `main` (see [below](#releases)), so in order to coordinate a release between the various products a `next` branch is used to accumulate changes and avoid unintentionally releasing part changes.
+
+Please read the following guidance:
+
+- Feature branches should be created from `next` and PRs made into `next`
+- If your change is noteworthy to our customers, update the [CHANGELOG.md](./CHANGELOG.md) in your PR with details of your changes
+  - You may need to create a new section if your change is the first since the last release
+- This is a public repo, so consider a squash-commit of your feature PRs to give a cleaner history
+- Create `integration/` branches from `next` if you want to keep a larger change away from the release during development
+- Don't forget to update the `docs/` directory with any noteworthy changes to functionality or configuration
+- Use internal communication channels to coordinate a release when you are ready
```

Comment:
As this is a public repo, should we clarify here that this is for SmartBear employees only?

For external contributions we'll ... <insert how we plan to support release coordination> (we can address this later perhaps).